### PR TITLE
[Peertube] Add playlist extractor and update instances list

### PR DESCRIPTION
**Pull request by u-spec-png**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [x] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

### CODE REVIEW

**Comment by pukkandan** [comment]
don't touch the existing list (including any broken instances). Just add any new ones to the top/bottom

**Comment by pukkandan** [comment]
also, best if u split this into 2 prs

**Comment by u-spec-png** [comment]
> also, best if u split this into 2 prs

Couldn't find any playlists on "old" instances.

**Comment by pukkandan** [comment]
> don't touch the existing list (including any broken instances). Just add any new ones to the top/bottom

this is what I meant https://github.com/yt-dlp/yt-dlp/commit/f60990ddfc779e84c784eb004c1047c768785452

File: yt_dlp/extractor/peertube.py [hunk 1]
```diff
@@ -1291,3 +1293,82 @@ def channel_data(field, type_):
...
+            video_title = try_get(video, lambda x: x['video']['name'])
+            yield self.url_result(
+                f'https://{host}/w/{shortUUID}', PeerTubeIE.ie_key(),
+                video_id=shortUUID, video_title=video_title)
+
+    def _real_extract(self, url):
+        mobj = self._match_valid_url(url)
+        host = mobj.group('host')
+        playlist_id = mobj.group('id')
+
```
Comment (pukkandan) [inline yt_dlp/extractor/peertube.py:1358]:
```suggestion
        host, playlist_id = self._match_valid_url(url).group('host', 'id')
```

**Pull request merged**